### PR TITLE
ELEMENTS-585: fix selectivity with non-balanced hierachical vocabulary

### PR DESCRIPTION
**Pull request by guirenard**

### CODE REVIEW

File: test/nuxeo-directory-widgets.html [hunk 1]
```diff
@@ -54,7 +55,119 @@
...
+                resolve(result.textContent);
+              } else if (timeout && (Date.now() - start) >= timeout) {
+                reject(new Error("timeout"));
+              } else {
+                setTimeout(waitForSuggestion.bind(this, resolve, reject), 30);
+              }
+            };
+            return new Promise(waitForSuggestion);
+          }
+          return flush().then(function() {
```
Comment (nelsonsilva) [inline test/nuxeo-directory-widgets.html]: nit: `flush().then(getSuggestedValue).then(....)`

**Comment by Gabez0r** [comment]
👍

**Comment by nelsonsilva** [comment]
👍

**Comment by afjusto** [comment]
👍

**Pull request merged**